site/src/components/image-switcher/ImageSwitcher.tsx

```diff
@@ -0,0 +1,40 @@
+import styles from "./ImageSwitcher.module.css";
+import { Switch } from "@salt-ds/lab";
+import { useState } from "react";
+import { Image } from "@jpmorganchase/mosaic-site-components";
+
+export interface ImageSwitcherProps {
+  label: string;
+  images: Array<{ alt: string; src: string }>;
+}
+
+export function ImageSwitcher(props: ImageSwitcherProps): JSX.Element {
+  const { label, images } = props;
+  const [toggle, setToggle] = useState(false);
+
+  const handleToggle = () => {
+    setToggle((old) => !old);
+  };
+
+  const currentImage = toggle ? images[1] : images[0];
+
+  return (
+    <div className={styles.container}>
```

Comment:
I wonder if we could use the (neutral) `<ExampleContainer>` here since it looks like image switcher uses the exact same border & padding. I.e.

```tsx
<ExampleContainer type="neutral">
  <div className={styles.header}>
  ...
</ExampleContainer>
```

Arguably, the image switcher is a specialised kind of example, so doing it that way also guarantees that `<ExampleContainer>` and `<ImageSwitcher>` will always share the same container styles, in case we ever change them in the future.

Reply:
I don't really see the need for this but if necessary it can be done at a later date.